values/velero/velero.gotmpl

```diff
@@ -16,7 +16,7 @@ resources:
 
 initContainers:
   - name: velero-plugin-for-{{ $provider }}
-    image: velero/velero-plugin-for-azure
+    image: digitalocean/velero-plugin:v1.1.0
```

Comment:
I know that this is just temporary but be sure to template this for multiple providers, I think we have azure, aws, do (digitalocean) and custom.

Reply:
Indeed. If a mapping needs to be created that resolves provider to image, then please make one in the top and just select it here. Example:

```yaml
{{- $pluginMap := dict "aws" "aws/velero-plugin:v1.1.0" "azure" "azure/velero-plugin:v1.1.0" "digitalocean" "digitalocean/velero-plugin:v1.1.0" }}
...
{{- $pluginImage := $pluginMap | get $v.cluster.provider "some/default-image:v1.1.0" }}
```

Reply:
For this first version, Jeho said it would be best to focus on Azure since that would be most relevant and time critical at this moment, and then extend further later. Definitely agree with you both!